docs/getting_started_tor.md

```diff
@@ -16,8 +16,7 @@ very simple Tor network:
 
 ```bash
 cd shadow/src/test/tor/minimal
-shadow --template-directory shadow.data.template tor-minimal.yaml > shadow.log
-./verify.sh
+./run.sh && ./verify.sh
```

Comment:
Maybe include a link to the source files so that people reading the documentation can see what's happening here?

Is there an advantage to having this `run.sh` script compared to running the test directly with `./setup test -- --build-config extra --label tor`?

Reply:
> Is there an advantage to having this run.sh script compared to running the test directly with ./setup test -- --build-config extra --label tor?

Maybe not. The main thing I want is a way for users to inspect the log files after running the experiment, which I think they can do using your setup script magic line. I didn't know you could run the Tor test that way, because I don't think it's documented anywhere or I missed it. The getting started tor markdown doc could be a good place to document it, rather than the custom run script.

Reply:
The closest documentation is https://github.com/shadow/shadow/blob/main/docs/developer_guide.md#extra-tests, which isn't tor-specific, but shows how to run the extra tests (which include the tor test).

Reply:
Done in e40070c